Sources/MongoSwift/MongoClient.swift

```diff
@@ -491,7 +521,16 @@ public class MongoClient {
         using connection: Connection? = nil,
         session: ClientSession? = nil
     ) throws -> T.OperationResult {
-        return try self.operationExecutor.execute(operation, using: connection, client: self, session: session)
+        return try self.operationExecutor.execute(operation, using: connection, client: self, session: session).wait()
+    }
+
+    /// Executes an `Operation` asynchronously using this `MongoClient` and an optionally provided session.
+    internal func executeOperationAsync<T: Operation>(
```

Comment:
eventually `executeOperation` above this will go away and this method will drop the `async`, but for now I've left the above so the types that are not yet async still compile.